content/fun/_index.md

```diff
@@ -7,6 +7,6 @@ weight = 7
 Fun commands and systems provide engaging ways to enrich your server environment.
 
 This chapter documents the fun systems which offer additional configuration via the control panel. To see all fun
-commands, [click here](/commands/all-commands#fun)
+commands, [click here](/core/all-commands#fun)
```

Comment:
```
The fun commands are documented in [their own section](/core/all-commands#fun) under all commands.
```
Might still be a tad weird but certainly better than "click here".
Point being, please reword this sentence such that it is easy to include a contextually precise hyperlink, as per:
https://ux.stackexchange.com/questions/12100/why-shouldnt-we-use-words-such-as-here-and-this-in-textlinks

Sorry for the weird comment layout, GitHub won't let me do the things I want to do.